docs/BREAKINGCHANGES.md

```diff
@@ -0,0 +1,217 @@
+Previously, when passing an empty format string to `-UFormat`, a somewhat unhelpful error message would appear.  A more descriptive error has been added.
+
+
+### Clean up console code [#4995](https://github.com/PowerShell/PowerShell/issues/4995)
+The following features were removed as they are not supported in PowerShell Core, and there are no plans to add support as they exist for legacy reasons for Windows PowerShell: psconsolefile switch and code, importsystemmodulues switch and code, and font changing code.
+
+
+### removed RunspaceConfiguration support [#4942](https://github.com/PowerShell/PowerShell/issues/4942)
+Previously, when creating a PowerShell runspace programmatically using the api you could use the legacy RunspaceConfiguration or the newer InitialSessionState.
+This change removed support for RunspaceConfiguration and only supports InitialSessionState.
+
+
+### CommandInvocationIntrinsics.InvokeScript bind arguments to $input instead of $args [#4923](https://github.com/PowerShell/PowerShell/issues/4923)
+An incorrect position of a parameter resulted in the args passed as input instead of as args.
+
+
+### remove unsupported -showwindow switch from get-help [#4903](https://github.com/PowerShell/PowerShell/issues/4903)
+`-showwindow` relies on WPF, which is not supported on CoreCLR.
+
+
+### allow * to be used in registry path for remove-item [#4866](https://github.com/PowerShell/PowerShell/issues/4866)
+Using `-literalpath` with an asterisk and the filesystem provider will now return an error rather than returning quietly.
+
+TODO: Comibine with 5197?
+
+
+### fix set-service failing test [#4802](https://github.com/PowerShell/PowerShell/issues/4802)
+`New-Service` should throw on System `-StartupType`
+TODO : Looks like this a change in test cases, not sure what the documentation process should be
+
+### Enhance the -split operator with negative maximum token counts to split from the end [#4765](https://github.com/PowerShell/PowerShell/issues/4765)
+After the enhancement, only a `<Max-substrings>` value of `0` will be accepted as the explicit signal that all tokens should be returned, however many are found in the input string(s).
+Negative `<Max-substrings>` values will work analogously to the already supported positive values, except that: they return the specified number of strings from the end of the string(s) and all individually extracted tokens are returned in input order, and whatever unsplit part remains, if any, is returned as the first token.
+
+
+### Rename $IsOSX to $IsMacOS [#4700](https://github.com/PowerShell/PowerShell/issues/4700)
+The naming in PowerShell should be consistent with our naming andd conform to Apple's use of macOS instead of OSX, however, for readability and consistently we are staying with Pascal casing.
+
+
+### Prepare for BOM-less UTF-8 default character encoding with respect to $OutputEncoding and console code page  [#4681](https://github.com/PowerShell/PowerShell/issues/4681)
```

Comment:
This is dupe of above as this is the issue and the other is the PR

Reply:
Removed